plugin/output/splunk/splunk.go

```diff
@@ -128,6 +131,7 @@ func (p *Plugin) out(workerData *pipeline.WorkerData, batch *pipeline.Batch) {
 		root := insaneJSON.Spawn()
 		root.AddField("event").MutateToNode(event.Root.Node)
 		outBuf = root.Encode(outBuf)
+		go insaneJSON.Release(root)
```

Comment:
Hmm... why it's in goroutine?
Better to spawn one root before the loop and reuse it.
Don't forget to clean it like `root.Decode("{}")` on each iteration.